packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/src/api/operations.ts

```diff
@@ -52,7 +53,23 @@ export function _publishCloudEventSend(
       contentType:
         (options.contentType as any) ??
         "application/cloudevents+json; charset=utf-8",
-      body: { event: event },
+      body: {
+        event: {
+          id: event["id"],
+          source: event["source"],
+          data: event["data"],
+          data_base64: uint8ArrayToString(
```

Comment:
The same concern with the default value for encoded string, why not just return undefined if `event["dataBase64"]`  is undefined.

Reply:
the same